Source/WebKit/UIProcess/ios/ProcessAssertionIOS.mm

```diff
@@ -387,9 +388,14 @@ - (void)assertion:(RBSAssertion *)assertion didInvalidateWithError:(NSError *)er
 void ProcessAssertion::acquireAsync(CompletionHandler<void()>&& completionHandler)
 {
     ASSERT(isMainRunLoop());
+
+    static Seconds acquireTimerMaxDuration = 2_s;
```

Comment:
2s isn't a very long time on a loaded system. In the problematic cases we've seen, the delay to take the assertion is forever, right?

If so, maybe this number can be much higher. 10s? 20s?

Reply:
Bumped to 20s